src/test/java/DukeTest.java

```diff
@@ -0,0 +1,60 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import duke.Deadline;
+import duke.Event;
+import duke.Parser;
+import duke.Task;
+import duke.Todo;
+
+
+
+public class DukeTest {
```

Comment:
Quite a few methods tested but probably the name of the Test file is a bit misleading since you are testing more of the tasks than the actual duke functionality. Probably renaming it to TaskTest would be more apt.